source/core/inject.js

```diff
@@ -41,6 +41,7 @@ var inject = function(_info, _window, _folder){
 	// Data - Options
 	tools.preloadScript('source/gca.data.js');
 	tools.preloadScript('source/gca.data.recipes.js');
+	tools.preloadScript('source/gca.data.recipeNames.js');
```

Comment:
This should not be loaded on every page.

Reply:
Move it to the appropriate section(s) on the `manager.js` instead; otherwise (better approach), dynamically load it using `gca_tools.load.script`, but the recipeNames.js will have tobe  moved to the resources folder.